### DISCUSSION TITLE
CF返回400，一个域名解析超过30个IP后解析失败，有解决办法吗？

### DISCUSSION THREAD

**Discussion by xuelang113** [jeessy2/ddns-go#565] [General]
我将DDNS-Go部署到CDN边缘节点服务器上，当服务器IP更换了，能自动解析。
域名托管在CF上，仅解析。在实际应用中，当一个二级域名解析超过30过IP时就显示请求不到记录。
我目前解析是数量48个IP，具体报错如下：
![image](https://user-images.githubusercontent.com/45958811/219936779-92884a57-1728-4982-85c1-936445f929d0.png)

**Comment by jeessy2** [comment]
这个应该是cf自身报错，有可能不支持。你可以咨询cf
